packages/cli/templates/webcomponents/igc-ts/projects/_base/files/package.json

```diff
@@ -31,33 +32,34 @@
     "igniteui-dockmanager": "~1.6.0",
     "@igniteui/material-icons-extended": "^2.10.0",
     "lit": "^2.0.2",
-    "typescript": "^4.5.2",
-    "babel-runtime": "^6.26.0"
+    "typescript": "^4.5.2"
   },
   "devDependencies": {
+    "@babel/preset-env": "^7.16.4",
+    "@open-wc/building-rollup": "^2.0.1",
+    "@rollup/plugin-babel": "^5.3.0",
+    "@rollup/plugin-node-resolve": "^13.0.6",
+    "rollup-plugin-copy-assets": "^2.0.3",
     "@open-wc/eslint-config": "^4.3.0",
     "@typescript-eslint/eslint-plugin": "^4.30.0",
     "@typescript-eslint/parser": "^4.30.0",
     "@web/dev-server": "^0.1.22",
+    "@web/rollup-plugin-html": "^1.10.1",
+    "@web/rollup-plugin-import-meta-assets": "^1.0.7",
+    "babel-plugin-template-html-minifier": "^4.1.0",
     "@web/test-runner": "next",
     "igniteui-cli": "~$(cliVersion)",
     "concurrently": "^6.3.0",
+    "deepmerge": "^4.2.2",
+    "rimraf": "^3.0.2",
+    "rollup": "^2.60.0",
+    "rollup-plugin-terser": "^7.0.2",
+    "rollup-plugin-workbox": "^6.2.0",
     "eslint": "^7.32.0",
     "tslib": "^2.3.1",
-    "@babel/cli": "^7.16.0",
-    "@babel/core": "^7.16.0",
-    "@babel/plugin-proposal-class-properties": "^7.16.0",
-    "@babel/plugin-transform-runtime": "^7.16.4",
-    "@babel/preset-env": "^7.16.4",
-    "@babel/preset-typescript": "^7.16.0",
-    "@types/source-map": "^0.5.7",
-    "babel-loader": "^8.2.3",
-    "babel-plugin-transform-custom-element-classes": "^0.1.0",
     "fork-ts-checker-webpack-plugin": "^6.4.2",
     "html-webpack-plugin": "^5.5.0",
```

Comment:
I suspect those are not needed after dropping the webpack build? Might not be a bad idea to check give all deps a once over.

PS: Technically the @open-wc/testing and typescript are dev deps, since they are not used in the final app, don't think it'll make much difference for an app build, but still.